### ISSUE TITLE
Additional page for assets like logos etc.

### ISSUE THREAD

**Issue by admdh** [postcss/postcss.org#210]
It would be nice to have page where everyone can download logos or some other useful assets.
As example - https://developer.uber.com/docs/design-guidelines

**Comment by marcustisater** [comment]
Definitely, you can look into the [brand repo](https://github.com/postcss/brand) for official logo.
